### PR TITLE
Make kubemark hollow node logging verbosity level configurable

### PR DESCRIPTION
**Pull request by shyamjvs**
cc @kubernetes/sig-scalability-misc @wojtek-t @gmarek

### CODE REVIEW

File: test/kubemark/start-kubemark.sh [hunk 1]
```diff
@@ -320,6 +320,8 @@ current-context: kubemark-context")
   sed -i'' -e "s/{{registry}}/${CONTAINER_REGISTRY}/g" "${RESOURCE_DIRECTORY}/hollow-node.json"
   sed -i'' -e "s/{{project}}/${PROJECT}/g" "${RESOURCE_DIRECTORY}/hollow-node.json"
   sed -i'' -e "s/{{master_ip}}/${MASTER_IP}/g" "${RESOURCE_DIRECTORY}/hollow-node.json"
+  sed -i'' -e "s/{{kubelet_verbosity_level}}/${KUBELET_TEST_LOG_LEVEL}/g" "${RESOURCE_DIRECTORY}/hollow-node.json"
```
Comment (gmarek) [inline test/kubemark/start-kubemark.sh]: `${KUBELET_TEST_LOG_LEVEL:---v=2}`

Comment (shyamjvs) [inline test/kubemark/start-kubemark.sh]: That happens already by default (Ref: https://github.com/kubernetes/kubernetes/blob/master/cluster/kubemark/gce/config-default.sh#L61)

**Comment by yujuhong** [comment]
@shyamjvs I don't think this works....see https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-kubemark-5-gce/7309/artifacts/kubemark-5-minion-group-f0cn/kubelet-hollow-node-dgcz4.log as an example. I still only see "Error" message in kubelet logs.

**Comment by wojtek-t** [comment]
@yujuhong - I think we also need this one: https://github.com/kubernetes/test-infra/pull/2058

**Pull request merged**